### PR TITLE
[DOCFIX] Remove docs

### PR DESCRIPTION
**Pull request by LuQQiu**
Remove dora unrelated docs

### CODE REVIEW

File: docs/en/administration/Basic-Logging.md [hunk 1]
```diff
@@ -222,100 +222,3 @@ commonOptions {
 }
 ) in 2 ms: alluxio.exception.status.NotFoundException: Path "/.DS_Store" does not exist.
 ```
-
```
Comment (Xenorith) [inline docs/en/administration/Basic-Logging.md:225]: wondering why remove these sections?

File: docs/en/administration/StressBench.md
Comment (Xenorith) [inline docs/en/administration/StressBench.md:1]: stressbench does not work on dora?

File: docs/en/contributor/Building-Alluxio-From-Source.md
Comment (Xenorith) [inline docs/en/contributor/Building-Alluxio-From-Source.md:1]: i feel we should still keep the basic parts of this and remove the integration related sections

File: docs/en/contributor/Code-Conventions.md
Comment (Xenorith) [inline docs/en/contributor/Code-Conventions.md:1]: isn't all this still relevant? why remove

**Review by Xenorith — CHANGES_REQUESTED** [review]
can run `bin/alluxio docGen` still? could do it in a follow up PR if not updated yet

**Comment by LuQQiu** [comment]
@Xenorith feel free to add back the sections you think it's useful
we are doing aggressive removing docs

**Comment by LuQQiu** [comment]
Stressbench relies on job-service which should be removed

File: docs/en/administration/Basic-Logging.md (hunk 1 above)
Comment (Xenorith) [reply docs/en/administration/Basic-Logging.md:225]: i leave this deleted

**Review by Xenorith — APPROVED** [review]

**Comment by Xenorith** [comment]
alluxio-bot, merge this please

**Pull request merged**